### DISCUSSION TITLE
Extend route option during import

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#548] [Q&A] [answered]
In kurviger.de you are able to extend a route - merging two routes to a single one. Here an example:
![2024-05-22 23_37_55-Buchbach - Ergolding _ Kurviger](https://github.com/devemux86/cruiser/assets/121806238/27acf65e-699c-4e8f-a836-a30eb9e88f94)
The first route was from START to (1) and the second was from (2) to END. The gap between (1) and (2) has been automatically routed and a new route has been created. 

Could you imagine to implement that in Cruiser import as well?

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the suggestion.

Import multiple routes as overlays is already available.

---

"Extend Route" will use the waypoints of both existing and imported route to calculate a new route.

**Comment by devemux86** [comment]
"Extend route" option during import will be available in the next Cruiser.

**Comment by devemux86** [comment]
"Extend route" option during import is available in **[Cruiser 5.1.0-b12](https://github.com/devemux86/cruiser/discussions/671)**.

Please test it well to see if everything works ok.

**Comment by devemux86** [comment, marked as answer]
- Extend route option during import is available in https://github.com/devemux86/cruiser/discussions/684
